drivers/net/virtio_net.c

```diff
@@ -1434,7 +1438,11 @@ static void virtnet_poll_cleantx(struct receive_queue *rq)
 		return;
 
 	if (__netif_tx_trylock(txq)) {
+#ifdef CONFIG_SECURITY_TEMPESTA
```

Comment:
We applying upstream patch, I guess we don't need  `CONFIG_SECURITY_TEMPESTA` here, especially if new kernel also contains this patch.

Reply:
I add it to have conflicts during applying our patches on the new kernel. To don't forget to not apply this patch